Improve Grafana notification tags

PetrSladek opened the pull request:
Added the ability to set global tags for Grafana annotation and the ability to set tags per Notification with templating capability

pasha-codefresh commented:
Hey @PetrSladek , could you please resolve conflicts and DCO check

pasha-codefresh reviewed (changes requested):
Please update documentation

Commit:
Improve Grafana notification tags

Add the ability to set global tags on the Grafana service configuration
(`tags` option) and per-notification tags with templating capability
(`grafana.tags` template field). Tags coming from the subscription
annotation, the service configuration, and the notification template are
merged together on the resulting Grafana annotation.

Document the new tags options in docs/services/grafana.md.

Signed-off-by: Petr Sládek <petr.sladek@team.blue>
